lib/features/family/features/reflect/bloc/grateful_cubit.dart

```diff
@@ -189,6 +189,7 @@ class GratefulCubit extends CommonCubit<GratefulUIModel, GratefulCustom> {
   }
 
   void _onEveryoneDonated() {
+    _reflectAndShareRepository.emptyAllProfiles();
```

Comment:
Is this method also called if people stop the game in between? (with the exit icon on the top)
Or should this be a reset thing that is called at the start of playing the game? :)